vortex/src/lib.rs

```diff
@@ -2,8 +2,8 @@ pub use vortex_array::*;
 pub use {
     vortex_buffer as buffer, vortex_datetime_dtype as datetime_dtype, vortex_dtype as dtype,
     vortex_error as error, vortex_expr as expr, vortex_file as file,
-    vortex_flatbuffers as flatbuffers, vortex_io as io, vortex_ipc as ipc, vortex_mask as mask,
-    vortex_proto as proto, vortex_sampling_compressor as sampling_compressor,
+    vortex_flatbuffers as flatbuffers, vortex_io as io, vortex_ipc as ipc, vortex_layout as layout,
+    vortex_mask as mask, vortex_proto as proto, vortex_sampling_compressor as sampling_compressor,
```

Comment:
Let me know if the layouts missing were intentional and I'll revert. I assumed it was forgotten since it was recently added.

Reply:
No it should be public